shared/actions/chat/index.js

```diff
@@ -955,6 +983,7 @@ function* chatSaga(): SagaGenerator<any, any> {
   yield Saga.safeTakeEvery('chat:updateBadging', _updateBadging)
   yield Saga.safeTakeEvery('chat:updateInboxComplete', _ensureValidSelectedChat, false, false)
   yield Saga.safeTakeEvery('chat:updateMetadata', _updateMetadata)
+  yield throttle(5000, 'chat:updateTyping', _updateTyping)
```

Comment:
Yeah, it was intentionally 5 seconds (typoed as 500ms in the PR description).  I'm going to drop it down to 2 seconds though.